README.md

```diff
@@ -31,172 +27,20 @@ Installation from GitHub:
-Please see the function docstring for more information.
-
-NOTE: The `fit` function will always default to using a
-GPU `backend` if available for the forward and backward passes
-during training of the LSTM.
-However, for the calulation of the average loss
-on the dataset after every epoch, you can decide
-if the CPU or GPU `backend` should be used (default is CPU).
-
-You can find an example usages of both `evotune` and `fit` [here][examples].
-
-If you want to pass a set of mLSTM and dense weights
-that were dumped in an earlier run,
-create params as follows:
-
-```python
-from jax_unirep.utils import load_params
-
-params = load_params(folderpath="path/to/params/folder")
-```
-
-If you want to start from randomly initialized mLSTM and dense weights instead:
-
-```python
-from jax_unirep.evotuning import init_fun
-from jax.random import PRNGKey
-
-_, params = init_fun(PRNGKey(0), input_shape=(-1, 10))
-```
-
-The weights used in the 10-dimensional embedding of the input sequences
-always default to the weights from the paper,
-since they do not get updated during evotuning.
-
-### UniRep stax
-
-We implemented the mLSTM layers in such a way that
-they are compatible with `jax.experimental.stax`.
-This means that they can easily be plugged into
-a `stax.serial` model, e.g. to train both the mLSTM
-and a top-model at once:
-
-```python
-from jax.experimental import stax
-from jax.experimental.stax import Dense, Relu
-
-from jax_unirep.layers import mLSTM1900, mLSTM1900_AvgHidden
-
-init_fun, apply_fun = stax.serial(
-    mLSTM1900(),
-    mLSTM1900_AvgHidden(),
-    Dense(512), Relu(),
-    Dense(1)
-)
-```
-
-Have a look at the [documentation][stax] and [examples][staxex]
-for more information about how to implement a model in `jax`.
-
 ## More Details
 
-To read more about how we reimplemented the model in JAX, we wrote it up.
-Both the [HTML](https://elarkk.github.io/jax-unirep/)
-and [PDF](https://elarkk.github.io/jax-unirep/paper.pdf)
-are available.
-
-## Feature Requests
-
-Feature requests are always welcome to be posted on the [issue tracker][it]!
-That said, please temper your expectations,
-as `jax-unirep` development happens as and when the lead maintainers
-(Arkadij Kummer and Eric Ma) encounter needs in their day jobs.
-We welcome your pull requests, and are happy to guide you through the development process
-and work with you to get what you need into the library,
-but any requests for us to implement features will be prioritized
-according to what we encounter in our day jobs.
+To read more about how we reimplemented the model in JAX,
+check out our preprint
+on [bioarxiv](https://www.biorxiv.org/content/10.1101/2020.05.11.088344v1).
 
-If you make in a pull request that gets accepted,
-we are more than happy to publicly acknowledge your contributions in the README
-and by sending tons of positive vibes throughout the Twitterverse and our LinkedIn connections!
+Docs are in the making, keep your eyes peeled!
```

Comment:
They should be ready right after merging now I think?

Reply:
Let's merge, we can edit that directly later haha. I can't wait to see the docs build on rtd.